test/parallel/test-abortcontroller.js

```diff
@@ -1,3 +1,4 @@
+/* global AbortSignal */
```

Comment:
The linter was complaining that `AbortSignal` is not defined (but it is). Not sure if I should fix this here, or somewhere else? 🤷

Reply:
That's definitely a bug, the fix is https://github.com/nodejs/node/pull/37693/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R318.

Reply:
Ah, I didn't scroll all the way to the bottom of that file. 😅

In that case, I'll wait for #37693 to land first. 🙂

Reply:
https://github.com/nodejs/node/pull/37693 has landed, you should be able to rebase and remove the eslint comment now.